lib/registry.js

```diff
@@ -43,14 +43,20 @@ class Registry {
 				}
 			}
 
-			let labels = '';
-			for (const key of Object.keys(val.labels)) {
-				labels += `${key}="${escapeLabelValue(val.labels[key])}",`;
-			}
-
 			let metricName = val.metricName || item.name;
-			if (labels) {
-				metricName += `{${labels.substring(0, labels.length - 1)}}`;
+
+			const keys = Object.keys(val.labels);
+			const size = keys.length;
+			if (size > 0) {
+				let labels = '';
+				let i = 0;
+				for (; i < size; i++) {
+					labels += `${keys[i]}="${escapeLabelValue(val.labels[keys[i]])}"`;
+					if (i !== size - 1) {
+						labels += ',';
+					}
```

Comment:
Almost. The exact way I like to do this is to loop up to `size - 1` and not have an `if` in the loop body (see comment I linked to previously). I think that will save a line also.